sweetalert2.d.ts

```diff
@@ -387,7 +391,7 @@ declare module 'sweetalert2' {
 
   type Awaited<T> = T extends Promise<infer U> ? U : T;
 
-  type SyncOrAsync<T> = T | Promise<T>;
+  type SyncOrAsync<T> = T | Promise<T> | Promisify<T>;
```

Comment:
I think we can inline `Promisify` here, since it's only used once, and probably will only ever be used once. This makes it easier to see what `SyncOrAsync` is. This also saves us needing to name it.

Reply:
Done.